EnglishFilter/sections/foreign.txt

```diff
@@ -6452,27 +6452,33 @@ zoomit.ir#$#body > nav.is-sticky[style="top: 130px;"] { top: 0!important; }
 zoomit.ir#%#document.cookie = "BnShowNum=4; path=/;";
 ! soft98.ir - Persian Blocker rules (daily fix)
 !#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android && !adguard_ext_android_cb)
-soft98.ir#$?#:xpath(/html/body/div[*]/div/div[*]/div/div[1]):has(a[href*="insta"]) { remove: true; }
+soft98.ir#%#//scriptlet("remove-class", "c6e1y2", ".c6e1y2", stay)
 soft98.ir#$?#[href*="linkdoni.soft98.ir"] { remove: true; }
-soft98.ir#$?#:xpath(/html/body/div[*]/div[*]/aside/section[*]/header) { remove: true; }
-soft98.ir#$?#:xpath(/html/body/div[*]/div[*]/aside/div/section/header) { remove: true; }
+soft98.ir#$#[src*="file.soft98.ir"]{color: #437593 !important; background-color: #437593 !important; border: #437593 !important;}
+soft98.ir#$#[src*="cdn.hostdl.com"]{color: white !important; background-color: white !important; border: white !important;}
+soft98.ir,~forum.soft98.ir#$?#:xpath(/html/body/div[*]/div/div[*]/div/div[1]):has(a[href*="98"]){ remove: true; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div/aside/section[*]/header){ remove: true; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div/div/section[*]/header){ remove: true; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div/aside/section[*]/div):has(img[src*="cdn.hostdl.com"]){ color: white !important; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div/aside/div/section/header){ remove: true; }
 soft98.ir#$?#:xpath(/html/body/div[*]/div/aside/div/section/div){ remove: true; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div/div/div/section){ remove: true; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div[*]/main/article[*]/span){ remove: true; }
+soft98.ir#$?#:xpath(/html/body/div[*]/div[*]/main/article[*]/div[*]):not(:has(img[src*="cdn.soft98.ir"])):not(:has-text(/)):not(:has(li)):not(:has(a[href*="/dl"])){ remove: true; }
 soft98.ir#$?#[href*="SMostafa.ir"] { remove: true; }
-||soft98.ir/engine/modules/antibot/antibot.php
-||soft98.ir/engine/ajax/controller.php
-||file.soft98.ir^$image,redirect=nooptext
-||cdn.hostdl.com^$image,redirect=nooptext
+||file.soft98.ir^$image,redirect=click2load.html
+||cdn.hostdl.com^$image,redirect=click2load.html
 ||linkdoni.soft98.ir^
 ||iranicard.ir^$domain=soft98.ir
+||iranicard.ir^$removeparam=/^utm_/
 ||asiatech.ir^$domain=soft98.ir
+||asiatech.ir^$removeparam=/^utm_/
 ||cdn.hostdl.com/goto/soft98
 ||sysban.ir^$domain=soft98.ir
 ||smostafa.ir^$domain=soft98.ir
 !#endif
 !#if (adguard_app_windows || adguard_app_mac || adguard_app_android)
-soft98.ir##div[id^="c"][id$="header"]
-soft98.ir###si2debar > section.card:not([id])
-soft98.ir###sideb3r2_s1ticky
```

Comment:
No need to remove these rules.

Reply:
@AdamWr [said](https://github.com/AdguardTeam/AdguardFilters/pull/126697#issuecomment-1209277507): _And it looks like some elements (header) which are not ads are also removed (by a rule with header)._

Reply:
Ok, added. But these depend on CSS names.

Reply:
I know, but they work fine in the app for now. Maybe we will come up with another solution for apps.